src/Api/CertificatesApi.php

```diff
@@ -477,14 +477,16 @@ public function generateAuthKey(string $product, string $csr): array
     }
 
     /** @see https://dm.yoursrs-ote.com/docs/api/ssl/resenddcv */
-    public function resendDcv(int $processId, DomainControlValidationCollection $domainControlValidationCollection): array
+    public function resendDcv(int $processId, ResendDcvCollection $resendDcvCollection): ?array
```

Comment:
This breaks backwards compatibility for this public method so we might need to take a closer look before releasing.